detox/local-cli/detox-build.js

```diff
@@ -2,14 +2,17 @@
 
 const _ = require('lodash');
 const program = require('commander');
+const {getConfigurationFile, getDefaultConfigurationFile} = require('../src/utils/getConfigurationFile');
 const path = require('path');
 const cp = require('child_process');
 program.description(`[convenience method] run the command defined in 'configuration.build'`)
-       .option('-c, --configuration [device configuration]', 'Select a device configuration from your defined configurations,'
-                                                             + 'if not supplied, and there\'s only one configuration, detox will default to it')
+  .option('--config-path [configPath]',
+    'Select a device config-file path, if not supplied, detox will default to the package.json, and if not found there, detox will fallback to .detoxrc')
+  .option('-c, --configuration [device configuration]', 'Select a device configuration from your defined configurations,' +
+    'if not supplied, and there\'s only one configuration, detox will default to it', getDefaultConfigurationFile())
```

Comment:
@oreporan , why `getDefaultConfigurationFile()`? Come on, it is a string like `android.emu.release`, `ios.sim.debug`, why require file? It is the second time I write it to you.